pep-0458.txt

```diff
@@ -337,12 +337,52 @@ paths of available target files together with their cryptographic hashes.
 can be accessed by the client. Each top-level role will serve its
-responsibilities without exception.  Figure 1 provides a table of the
+responsibilities without exception.  Table 1 provides an overview of the
 roles used in TUF.
 
-.. image:: pep-0458-1.png
-
-Figure 1: An overview of the TUF roles.
++-----------------------------------------------------------------------------+
+|                          Roles and Responsibilities                         |
++-----------------+-----------------------------------------------------------+
+|       root      | The root roles is the locus of trust for the entire       |
+|                 | repository. The root role signs the root.json metadata    |
+|                 | file. This file indicates which keys are authorized for   |
+|                 | each of the top-level roles, including for the root role  |
+|                 | itself. The roles "root", "snapshot", "timestamp" and     |
+|                 | "targets" must be specified and each has a list of public |
+|                 | keys.                                                     |
++-----------------+-----------------------------------------------------------+
+|     targets     | The targets role is responsible for indicating which      |
+|                 | target files are available from the repository. More      |
+|                 | precisely, the targets role shares the responsibility of  |
+|                 | providing information about the content of updates. The   |
+|                 | targets role may also delegate trust of repository files  |
+|                 | to other roles (delegated roles).                         |
++-----------------+-----------------------------------------------------------+
+| delegated roles | If the top-level targets role performs delegation, the    |
+|                 | resulting delegated roles can then provide their own      |
+|                 | metadata files. The format of the metadata files provided |
+|                 | by delegated targets roles is the same as that of         |
+|                 | targets.json. As with targets.json, the latest version of |
+|                 | metadata files belonging to delegated roles are described |
+|                 | in the snapshot role's metadata.                          |
++-----------------+-----------------------------------------------------------+
+|     snapshot    | The snapshot role is responsible for ensuring that        |
+|                 | clients see a consistent repository state. It provides    |
+|                 | repository state information by indicating the latest     |
+|                 | versions of all metadata files on the repository in       |
+|                 | snapshot.json. The only metadata file not listed in       |
+|                 | snapshot.json is timestamp.json, which is always created  |
+|                 | after snapshot.json.                                      |
```

Comment:
This needs to be updated to the latest version of the spec. The current spec does not have root in snapshot either.